ci/mac_ci_setup.sh

```diff
@@ -26,6 +26,5 @@ do
     is_installed "${DEP}" || install "${DEP}"
 done
 
-# /Applications/Xcode.app/Contents/Developer
+sudo xcodebuild -runFirstLaunch
```

Comment:
this needs to be after the switch, assuming the switch is actually working

Reply:
Updated, looks like we're still getting the following error:

> xctestrunner.shared.ios_errors.IllegalArgumentError: The simulator os version 12.4 is not supported. Supported simulator os versions are [u'8.4', u'9.0', u'9.1', u'9.2', u'9.3', u'10.0', u'10.1', u'10.2'].

The Xcode path is still indicating 10.3, though:

> /Applications/Xcode_10.3.app/Contents/Developer

And iPhone X on 12.4 is listed as being available:

```-- iOS 12.4 --
    iPhone 5s (22DA5C87-561D-4147-A2ED-62F6C88C3690) (Shutdown) 
    iPhone 6 (9B3290A6-1E5F-4ACA-832D-C327C93A3A28) (Shutdown) 
    iPhone 6 Plus (29B6D028-AA03-481F-926E-0CAB108C3B1E) (Shutdown) 
    iPhone 6s (8841751E-A7FB-420C-9C6A-00C7646E120D) (Shutdown) 
    iPhone 6s Plus (57CEE6E3-0F66-42CD-A9C1-0212BA92522A) (Shutdown) 
    iPhone 7 (7852BB53-4751-4B0D-B0E7-2D5CC82FB2E5) (Shutdown) 
    iPhone 7 Plus (62FE3DA8-7C06-4353-AA46-02C50D0574ED) (Shutdown) 
    iPhone 8 (F4E5A2DE-1609-4C86-8AC4-0B7D799BDF78) (Shutdown) 
    iPhone 8 Plus (33DCB7FA-E504-4BC1-812A-82CD2E16CBEC) (Shutdown) 
    iPhone SE (6A941539-C0FD-4396-965C-3BC164B42840) (Shutdown) 
    iPhone X (067FE3A9-9FBB-4309-ACAA-5C6CBC6CA06F) (Shutdown) ```